core/libs/editor/src/lib/editor-dialog/components/fields/base/see-input-field/see-input-field.component.html

```diff
@@ -20,18 +20,19 @@
 >
   <mat-label for="see">see</mat-label>
   <mat-chip-grid [formControl]="chipControl" #chipList>
-    <mat-chip-row
-      *ngFor="let element of elements"
-      [matTooltip]="element.urn"
-      [attr.data-cy]="'chip__' + (element.name || element.urn)"
-      (removed)="removeElement(element)"
-      matTooltipClass="mat-tooltip--long"
-    >
-      <span class="chip-content">{{ element.name || element.urn }}</span>
-      <button data-cy="see-remove-chip" matChipRemove>
-        <mat-icon>cancel</mat-icon>
-      </button>
-    </mat-chip-row>
+    @for (element of elements; track element) {
```

Comment:
In this case I would track by `urn`, is the object `element` is not changing the reference. Please take a look on the others `@for` since there are more examples like this one